algorithms/a3c/model.py

```diff
@@ -111,3 +132,94 @@ def forward(self, inputs):
         x = hx
 
         return self.critic_linear(x), self.actor_linear(x), (hx, cx)
+
+
+class A3C_LSTM_GA(torch.nn.Module):
+    """
```

Comment:
It would be nice to add an ASCII art diagram of the architecture. Something like:
```
       ________
cx--|   LSTM    | --- Critic -- Q
hx--|________|
```
Obviously my example is wrong, but just an idea

Reply:
Also applicable to A3C above

Reply:
hahaha, might be worth it but it might take me 15 minutes haha